pages/admin/class/edit/[id].tsx

```diff
@@ -349,7 +349,7 @@ export default function CreateClass({ session }: Props): JSX.Element {
                 ></CheckBoxField>
                 {EDIT ? (
                     <Button
-                        key={className} //When loading from localstorage finishes this causes the button to re-render
+                        key={"Edit Class_" + className + "_" + price + "_" + programs.length}
```

Comment:
that's a pretty beefy key 😳

Reply:
Doesn't look like this fixed the problem. The troublesome field we're looking at here is the Teacher select specifically. I don't think tweaking the key here will help.

![image](https://user-images.githubusercontent.com/44826218/145701198-ac5d7fe2-aaa4-4d34-9ff7-030438b67ec6.png)